_posts/2021-03-19-new-ssl-features.adoc

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "Upcoming SSL features: Multiple certificate revocation lists support and SSLv2Hello support"
+date: 2021-03-19
+tags: wildfly elytron
+synopsis: An overview of the upcoming SSL/TLS features including the ability to configure
+multiple certificate revocation lists and SSLv2Hello support.
```

Comment:
Just tried checking what the blog post looks like and the multi-line synopsis and the colon in the title cause some issues when rendering. I need to create an aggregate PR to fix some dates in other posts so will fix these issues there.